conf/application.conf

```diff
@@ -346,3 +348,18 @@ mongodb {
 
 # must be a Base64 encoded string of 128/192/256 bits
 crypto.encryption-key = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+
+
+health {
+
+  nsi-connection {
+    enabled = true
+    poll-period = 1 minute
+    minimum-poll-period = 1 minute
+    poll-timeout = 30 seconds
```

Comment:
Pls set - following a late Fri meeting - this to 50s, as NS&I's longest timeout has been set to 40s and MDTP is 60s